src/pages/_document.tsx

```diff
@@ -7,7 +7,7 @@ class MyDocument extends Document {
         <Head>
           <link
             rel="stylesheet"
-            href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap"
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&display=swap"
```

Comment:
Loading in 400, 700 and 900 for now.

400 (regular) for normal body text, 700 (bold) for emphasis, and 900 (black) for our hero/big text.